simdistserve/base/request.py

```diff
@@ -61,6 +63,11 @@ def __init__(
         # set this value if a request belongs to a particular chunk
         # The last worker in the pipeline unset this value at a chunk's end.
         self.chunk_id = None
+        # after the request is finished prefill, `kvcache_generated` should be set to `True`.
+        self.kvcache_generated = False
+        self.prefill_is_done = False
+        self.kvcache_is_transferred = False
+        self.prefill_worker = None
```

Comment:
Is this one prefill worker? Just want to make sure we free the kv from every prefill worker that processed this request (consider PP)

Reply:
solved this